lib/rubocop/cop/style/file_touch.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # Checks for usage of `File.open` in append mode with empty block.
+      #
+      # @safety
```

Comment:
Just leaving this here as quick feedback. `FileUtils`, unlike `File`, is not loaded by default and must be required:

```
$ ruby -e 'FileUtils'
-e:1:in `<main>': uninitialized constant FileUtils (NameError)

FileUtils
^^^^^^^^^
Did you mean?  FileTest
```

Not sure if there is a cop that inserts a require somewhere.

Reply:
True, but this then affects [Lint/NonAtomicFileOperation](https://docs.rubocop.org/rubocop/cops_lint.html#lintnonatomicfileoperation) as well 🤔 

I think the reason for its usage might be that in a bundler context `FileUtils` will always be required, so it's usually considered present.